process/coordinator/process.go

```diff
@@ -799,9 +802,17 @@ func (tc *transactionCoordinator) CreatePostProcessMiniBlocks() block.MiniBlockS
 	return miniBlocks
 }
 
+func (tc *transactionCoordinator) addGasForScheduled(gasAndFees scheduled.GasAndFees) {
+	tc.gasHandler.SetGasRefunded(gasAndFees.GasRefunded, []byte(prevScheduledTxs))
+	tc.gasHandler.SetGasConsumed(gasAndFees.GasProvided, []byte(prevScheduledTxs))
+	tc.gasHandler.SetGasPenalized(gasAndFees.GasRefunded, []byte(prevScheduledTxs))
```

Comment:
gasAndFees.GasPenalized

An unit test for CreateBlockStarted could be added to validate the addGasForScheduled

Reply:
ok